net/ostiary/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ostiary
+PKG_VERSION:=4.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0-only
+PKG_MAINTAINER:=Chris Geraghty <cgretski@hotmail.com>
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/ostiary-$(PKG_VERSION)
+PKG_SOURCE:=ostiary-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://ingles.homeunix.net/software/ost/latest/
+PKG_HASH:=1b6a0a8a17fd3aa0f6511cdda81558d54f11023a0c953201ddaaef35f56a82c4
+PKG_CAT:=zcat
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ostiary
+  SECTION:=base
+  CATEGORY:=Network
+  TITLE:=Secure remote execution utility
+  URL:=http://ingles.homeunix.net/software/ost/index.html
+endef
+
+define Package/ostiary/description
+ Replay-resistent predefined remote-command trigger  
+endef
+
+define Package/ostiary/conffiles
+/etc/ostiary.cfg
+/etc/init.d/ostiaryd
+endef
+
+define Build/Compile
```

Comment:
Remove

Reply:
Hi @diizzyy 
Please clarify what needs removing.

The Build/Compile override is necessary as make requires the target "all" ( there is no default target)

Reply:
Ahh sorry, I looked at the Makefile a bit too fast...
This is probably a better idea however as it avoids man pages etc?
https://gitlab.labs.nic.cz/turris/openwrt-packages/blob/74d668f29ecb2b407e9976ce8c3391fd2beb934e/admin/ostiary/Makefile#L42

Reply:
The main differences I can see from the old package are the postinst and prerm sections (which I believe are duplicating default behavior);  I'm not including the ssh_enable/disable scripts which were custom to the old maintainers environment, and I'm not putting the single config file in it's own subdirectory.

The make all recipe doesn't (appear to) touch the man pages, just the binaries.
The only files packaged are:
./etc/init.d/ostiary
./etc/ostiary.cfg
./lib/upgrade/keep.d/ostiary
./usr/bin/ostclient
./usr/bin/ostiaryd

This is my first attempt to create a package (and use github's pull-requests);  please let me know if there's any other advantage of the old Makefile that I'm missing or any other best-practice/standards violations I should be aware of.

Thanks

Reply:
Also, do I need to list any dependencies, the only deps. are libgcc and libc  which are already dependencies of base-files.
The binaries link as follows:
/lib/ld-musl-mips-sf.so.1 (0x77ecc000)
        libgcc_s.so.1 => /lib/libgcc_s.so.1 (0x77e8f000)
        libc.so => /lib/ld-musl-mips-sf.so.1 (0x77ecc000)